### DISCUSSION TITLE
Automatically replace colons with hyphens

### DISCUSSION THREAD

**Discussion by dajana91** [aclap-dev/video-downloadhelper#271] [Help and Q&A]
### Your question

Hi,

How is it possible to rename colons to hyphens when naming files? Example: You download a video using a page title, e.g. "Example: Great Site". The downloaded file should be "Example - Great Site" then. So in this case the : is replaced with - AND in addition there is a space before the -, otherwise the file would be called "Example- Great Site", what would look bad.

I tried with RegEx, but it did not work: Any idea? Or maybe a feature request?

### Addon & CoApp version and details

Video DownloadHelper 8.2.2.5a1 (beta) at the moment.

**Comment by paulrouget** [comment]
Yeah, that looks bad. I'll try to improve the situation in the next release.

**Comment by dajana91** [comment]
That would be so great! Where can I see if the feature is built in?

**Reply by dajana91** [reply]
... Hello? @paulrouget

**Comment by dajana91** [comment]
Still nothing?
